src/main/java/org/cactoos/text/BytesAsText.java

```diff
@@ -114,4 +114,9 @@ public String asString() throws IOException {
         return new String(this.bytes.asBytes(), this.charset);
     }
 
+    @Override
+    public int compareTo(final Text text) {
+        return new UncheckedText(text).compareTo(this);
```

Comment:
@alex-semenyuk We need wrap `text` into `UncheckedText` to not throws an exception and after that, call `Unchecked`s `compareTo` to compare itself. How all `Text` implementations are immutable, I don't see any problem.  The reason it's because `UncheckedText` has the real comparing between strings and to avoid to replicate this code I did this way. But I'm opened to suggestions! :)